policy_packs/aws/ssm/enforce_install_packages_on_instances_using_ssm_package_installer/README.md

```diff
@@ -0,0 +1,163 @@
+---
+categories: ["security"]
+primary_category: "security"
+---
+
+# Enforce Install Packages On Instances Using AWS SSM Package Installer
+
+Enforcing the installation of packages on instances using the AWS SSM Package Installer is crucial for maintaining a consistent, secure, and automated software deployment process. This measure ensures that all package installations are managed and logged through AWS Systems Manager, enhancing security, compliance, and operational efficiency by providing centralized control and monitoring of software installations.
+
+This [policy pack](https://turbot.com/guardrails/docs/concepts/resources/smart-folders) is designed to allow package installation on instances with or without direct access to original package source and can help you configure the following settings for SSM documents:
+
+- Set the region ARN where the SSM resources will be deployed
+- Set the IAM role ARN to be used by SSM to run documents
+- Create three SSM documents in the specified region
+  - Turbot-LinuxPackageInstaller-PublicNetwork
+  - Turbot-LinuxPackageInstaller-PrivateNetwork
+  - Turbot-LinuxS3PackageInstaller
+
+## Documentation
+
+- **[Review Policy settings →](https://hub-guardrails-turbot-com-git-development-turbot.vercel.app/policy-packs/enforce_install_packages_on_instances_using_ssm_package_installer/settings)**
+- Using the SSM documents
+  - There are two entry point types:
+    - Turbot-LinuxPackageInstaller-PublicNetwork
+    - Turbot-LinuxPackageInstaller-PrivateNetwork
+
+### Turbot-LinuxPackageInstaller-PublicNetwork
```

Comment:
Why are these under the documentation section? A follow-up question is what am I meant to do with this information?

Reply:
This was added for informational purpose on what the resources would be used for once created. Felt like it was useful information and copied directly from the existing readme for this stack example.